.gitlab/ci/sample_code_windows.gitlab-ci.yml

```diff
@@ -0,0 +1,168 @@
+getting_started_windows:
+  extends: .windows:defaults
+  stage: test
+  tags:
+    - shared-windows
+    - windows
+    - windows-1809
+  before_script:
+    - Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
+    - choco install -y python --version=3.8.9
+    - RefreshEnv
```

Comment:
Chocolatey comes with all GitLab Windows instances by default, right? If so, I see no reason to not benefit from it.

Reply:
What I"m wrestling with for this is since it's trying to replicate the sample I want it to be as close to the Sample Code as possible. 

Leaving this as is seems correct for now! Irks me a bit to have this code seperate from the docs but this seems fine